recaf-ui/src/main/java/me/coley/recaf/ui/control/code/bytecode/AssemblerArea.java

```diff
@@ -295,10 +292,10 @@ private VirtualizedContextMenu<Suggestion> createSuggestionsMenu(int position, G
 					&& e.getInputEvent() instanceof KeyEvent
 					&& ((KeyEvent) e.getInputEvent()).getCode() == KeyCode.TAB) {
 				replaceText(suggestionGroup.start().getStart() + 1, suggestionGroup.start().getEnd() + 1,
-						TextDisplayUtil.escapePath(suggestionText));
+						EscapeUtil.escape(suggestionText));
```

Comment:
Was renamed to `shortenEscape`.

Should use `TextDisplayUtil` for consistency of using it when possible.

Reply:
What method should I use? As this use case needs to have a path compliant with ASM language.

Reply:
For the text to be compliant with the input transformer all whitespaces need to be formatted which the method `EscapeUtil.escapeSpace` took care of this, but because we take the liberty of saying that we never need to deal with obfuscated input / malformed names.

Reply:
> As this use case needs to have a path compliant with ASM language.

Ah..... yeah. This is correct then.